CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ New configuration to access the database:
 - `EDC_SERVER_DB_CONNECTION_POOL_SIZE`
   - The property controls the maximum size that the pool is allowed to reach, including both idle and in-use connections. Basically this value will determine the maximum number of actual connections to the database backend.
   - Defaults to `3`
+- `MY_EDC_DATASOURCE_PLACEHOLDER_BASEURL`
```

Comment:
this should not pop up in the changelog, as EDC CE users should not have to re-configure anything

Reply:
How will they know that something must be updated?
I must know under which URL they show their EDC and I can't guess...

If I don't mention this, the placeholder data source will not be accessible.

Reply:
Also, about not writing this kind of stuff here:

In the EE release procedure, we have this point
https://github.com/sovity/edc-ee/blob/main/.github/ISSUE_TEMPLATE/release.md?plain=1#L35

And not writing this kind of change in the CE will prevent whoever does the releasing to gather that there is now a new config parameter, optional in CE, most likely mandatory in EE.